lib/components/viewers/styled.js

```diff
@@ -1,18 +1,58 @@
-import styled from 'styled-components'
+import styled, { keyframes } from 'styled-components'
 
 /** Route Details */
-export const Container = styled.div`
-  background-color: ${(props) =>
-    props.full ? props.backgroundColor || '#ddd' : 'inherit'};
-  color: ${(props) => (props.full ? props.textColor : 'inherit')};
-  height: 100%;
-  overflow-y: hidden;
-`
 
 export const RouteNameContainer = styled.div`
   padding: 8px;
   background-color: inherit;
 `
+
+const backgroundColorTransition = keyframes`
+    from {
+      background-color: white;
+    }
+`
+const fade = keyframes`
+    0% { opacity: 0%; }
+    100 { opacity: 100%; }
+`
+const fadeDelay = keyframes`
+0% { opacity: 0%; }
+50% { opacity: 0% }
+100 { opacity: 100%; }
+`
+
+export const StyledRouteViewerContainer = styled.div`
+  animation: ${(props) => props.patternView && backgroundColorTransition} 150ms
```

Comment:
This sounds like it should be a `transition` attribute on `background-color`, the original state being the white of the original render and the ending state being the background color of the route. Also, `transition` includes a delay parameter that can replace the delay emulation you added in the JS code.

Reply:
Good call! Not sure why I jumped straight to `keyframes`, `transitions` work just as well. Thanks!